src/pint/models/astrometry.py

```diff
@@ -494,7 +494,11 @@ def barycentric_radio_freq(self, toas):
         tbl = toas.table
         L_hat = self.ssb_to_psb_xyz_Eclip(epoch=tbl["tdbld"].astype(numpy.float64))
         v_dot_L_array = numpy.sum(tbl["ssb_obs_vel_ecl"] * L_hat, axis=1)
-        return tbl["freq"] * (1.0 - v_dot_L_array / const.c)
+        ans = tbl["freq"] * (1.0 - v_dot_L_array / const.c)
+        f = open("freq.out", "a")
```

Comment:
Oops, looks like some debugging lines got included in this commit. :-)

Reply:
Ah, my apologies! I will get this right eventually...

Reply:
No worries!

Just a note that printing to a file works, but may not be the easiest way to debug. There are many ways, of course, but if I want to inspect some variables at a particular point, sometimes I just put a "raise" in the code to generate an exception. If I then run the code in ipython when it hits the exception, I just type "%debug" and it puts me in the debugger. There I can view and print all the objects that are in scope and typing "up" moves up one in the call stack. Can be very handy!